docs/StorageClass.md

```diff
@@ -73,6 +88,9 @@ This will mount local, ephemeral volumes for `/opt/splunk/etc` and
 forever whenever the Pod is removed from a node (for any reason). We strongly
 discourage and do not support using this for any production environments.*
 
+## Storage Space Considerations
+
+Please see Storage guidelines at https://github.com/splunk/splunk-operator/blob/develop/docs/CustomResources.md 
```

Comment:
I don't see the intent here. What would a reader learn on that page that isn't already defined here?

Reply:
Removed this paragraph and added SmartStore para in the Readme for a better access to the smartstore feature benefit